src/core/api_client/apiClient.ts

```diff
@@ -252,11 +251,15 @@ export class ApiClient {
       errorMessage = ApiClient.parseError(problem);
     }
 
-    if (!errorsConfig || !errorsConfig.dontShowErrors) {
-      this.notificationService.error({
-        title: this.translationService.t('error'),
-        description: this.translationService.t(errorMessage),
-      });
+    if (problem.kind !== 'canceled') {
+      if (!errorsConfig || !errorsConfig.dontShowErrors) {
```

Comment:
My suggestion to make this check more readble
```ts
if (!errorsConfig?.dontShowErrors) {
```

Reply:
One more suggestion)
```ts
if (problem.kind === 'canceled') {
  console.debug('Request was canceled');
} else if (!errorsConfig?.dontShowErrors) {
  this.notificationService.error({
    title: this.translationService.t('error'),
    description: this.translationService.t(errorMessage),
  });
}
```

Reply:
Love it, i make it event more readable after thinking a while